ambry-router/src/main/java/com.github.ambry.router/PutOperation.java

```diff
@@ -860,7 +899,6 @@ void onErrorResponse(ReplicaId replicaId) {
      * @param exception the exception that may be set as the chunkException.
      */
     private void setChunkException(RouterException exception) {
-      // @todo: update error metric.
```

Comment:
Effectively, yes. This gets accounted as part of the request errors to each datanode. I am also adding a general error request error metric as part of this patch.